bahmnicore-omod/src/main/resources/liquibase.xml

```diff
@@ -447,7 +468,7 @@
             set @answer_concept_id = 0;
             call add_concept(@concept_id, @concept_name_short_id,
             @concept_name_full_id,'REFERRED_OUT','REFERRED_OUT', 'N/A', 'Misc', true);
-            select @labresults_concept_id := concept_id, min(concept_id) from concept_name where name = 'LABRESULTS_CONCEPT';
+            select @labresults_concept_id := concept_id, min(concept_id) from concept_name where name = 'LABRESULTS_CONCEPT' GROUP BY concept_id;
```

Comment:
@samuelmale what what the issue on this one?

Reply:
Without the `GROUP BY` clause, mysql throws this error.

`ERROR 1140 (42000): In aggregated query without GROUP BY, expression #1 of SELECT list contains nonaggregated column 'bahmni-4.concept_name.concept_id'; this is incompatible with sql_mode=only_full_group_by
`

Reply:
:heavy_check_mark:

Reply:
@samuelmale , same thing here, this changes the checksum of the `changeSet`. Must create assign a new changeSet ID.